source/_integrations/livisi.markdown

```diff
@@ -2,14 +2,16 @@
 title: LIVISI SmartHome
 description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
 ha_category:
+  - BinarySensor
   - Switch
 ha_iot_class: Local Push
-ha_release: 2022.12.0
+ha_release: 2023.01.0
```

Comment:
This should not change, as this is the original release the integration was added (not last updated).